tests/org.eclipse.xtend.ide.tests/xtend-gen/org/eclipse/xtend/ide/tests/contentassist/Bug365368Test.java

```diff
@@ -19,7 +19,7 @@
   @Test
   public void testEscapedIdentifierNoPrefix() throws Exception {
     ContentAssistProcessorTestBuilder _newBuilder = this.newBuilder();
-    ContentAssistProcessorTestBuilder _append = _newBuilder.append("class C {\n\t\t\tdef void ^as() {}\n\t\t\tdef void aa() {}\n\t\t\tdef void bb() {}\n\t\t\tdef void m() { this.");
+    ContentAssistProcessorTestBuilder _append = _newBuilder.append("class C {\r\n\t\t\tdef void ^as() {}\r\n\t\t\tdef void aa() {}\r\n\t\t\tdef void bb() {}\r\n\t\t\tdef void m() { this.");
```

Comment:
I used to use the other option (keep them as-is). But Oomph uses autocrlf as the default. That's because it's considered the standard option for git users on Windows.